internal/apis/controlplane/kubeadm/v1alpha3/conversion.go

```diff
@@ -82,6 +82,10 @@ func (src *KubeadmControlPlane) ConvertTo(dstRaw conversion.Hub) error {
 		}
 		dst.Spec.KubeadmConfigSpec.JoinConfiguration.Patches = restored.Spec.KubeadmConfigSpec.JoinConfiguration.Patches
 		dst.Spec.KubeadmConfigSpec.JoinConfiguration.SkipPhases = restored.Spec.KubeadmConfigSpec.JoinConfiguration.SkipPhases
+
```

Comment:
Same as above here with *File and we should also get rid of the fuzzer in conversion_test.go (+ same for v1alpha4)